.github/scripts/generate_ci_workflows.py

```diff
@@ -147,37 +147,37 @@ def generate_workflow_file(self, workflow_template: jinja2.Template) -> None:
 
 
 WINDOWS_WORKFLOWS = [
```

Comment:
Not doing the same thing for windows as the logic is essentially the same if not easier.